exercises/pascals-triangle/src/example.h

```diff
@@ -0,0 +1,7 @@
+#ifndef PASCALS_TRIANGLE_H
+#define PASCALS_TRIANGLE_H
+#include <stddef.h>
+
+int pascals_triangle(int count, size_t input[][count]);
```

Comment:
You never use the return value for this function. Either make the return value a void or change the test suite. (You already use the array as a return/error condition).

As a nitpick, you should rename `input` to be something like `result`, since it's not actually and input.